.github/workflows/e2e.yaml

```diff
@@ -0,0 +1,220 @@
+name: e2e
```

Comment:
@BenTheElder @jkh52 do you know why the repo does not run the actions? do we have to enable them somewhere?

Reply:
@jkh52 you have admin privileges in the repo https://github.com/kubernetes/org/blob/08dbac9ce837641b11cda031977364d94aa0258f/config/kubernetes-sigs/sig-cloud-provider/teams.yaml#L8

Can you go to the Settings field and enable github actions https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/enabling-features-for-your-repository/managing-github-actions-settings-for-a-repository#managing-github-actions-permissions-for-your-repository

or do you want me to get admin permissions and enable them?

Reply:
I want to have CI to validate on presubmit ipv4, ipv6 and dual stack, since we don't have it anywhere

Reply:
ok, let me change the path

Reply:
Agreed, actions seem enabled (but I'm not experienced with them). We can look together if it is not working.

Reply:
we can check later why it does not work, but it does not seem a problem to merge it by now